lib/beaker/host_prebuilt_steps.rb

```diff
@@ -409,10 +409,14 @@ def disable_updates hosts, opts
     def enable_root_login host, opts
       logger = opts[:logger]
       block_on host do |host|
-        logger.debug "Update /etc/ssh/sshd_config to allow root login"
+        logger.debug "Update sshd_config to allow root login"
         if host['platform'] =~ /osx/
-          host.exec(Command.new("sudo sed -i '' 's/#PermitRootLogin no/PermitRootLogin Yes/g' /etc/sshd_config"))
-          host.exec(Command.new("sudo sed -i '' 's/#PermitRootLogin yes/PermitRootLogin Yes/g' /etc/sshd_config"))
+          # If osx > 10.10 use '/private/etc/ssh/sshd_config', else use '/etc/sshd_config'
+          ssh_config_file = '/private/etc/ssh/sshd_config'
+          ssh_config_file = '/etc/sshd_config' if host['platform'] =~ /^osx-10\.[1-9]{1}[0]{0,1}$/i
```

Comment:
@wilson208 I'm not sure if this regex is working as intended; also, it seems to me that it could be written a bit clearer. How does the following regex look to you? 

````
^osx-10\.(9|10)
````

As far as I can tell, the regex you have won't ever match because of the `$` at the end means that the string terminates, but host platform strings still have an architecture type appended after the version.